core/client/fs/src/main/java/alluxio/client/file/AlluxioFileInStream.java

```diff
@@ -233,6 +234,11 @@ private int positionedReadInternal(long pos, byte[] b, int off, int len) throws
       return -1;
     }
 
+    LOG.info("AMDEBUG pread len {} ", len);
+    if (len < 2 * Constants.MB) {
```

Comment:
I think it is good to have our heuristic to cover the typical case, but should we check if it has been set to allow the user to override?

Reply:
yep, i'll make this a client config which can be overridden by the user